core/src/main/java/org/lflang/TargetConfig.java

```diff
@@ -383,7 +383,9 @@ public static class PlatformOptions {
     /**
      * The string value used to determine what type of embedded board we work with and can be used
      * to simplify the build process. For example, when we want to flash to an Arduino Nano 33 BLE
-     * board, we can use the string arduino:mbed_nano:nano33ble
+     * board, we can use the string arduino:mbed_nano:nano33ble TODO: move to lingo Can also be used
```

Comment:
We generally want to move stuff like this to `lingo`, so let's not leave arbitrary `TODOs` related to this in the code.